app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
```

Comment:
We should add meaningful naming to our variable. For example: `no_mask_friends_count` or `masks_to_buy` from requirements should be better options.
It is important to give meaningful names. Imagine huge codebase with many variables which have naming: `total`, `amount`, `count` etc. It's so much easier just to read name of variable and understand it's purpose.